policyengine_us/parameters/gov/states/ok/tax/income/agi/subtractions/military_retirement/amount.yaml

```diff
@@ -0,0 +1,19 @@
+description: Oklahoma limit on military retirement benefit AGI subtraction per person.
```

Comment:
How should I name this parameter? It's neither a maximum amount nor a minimum amount.